File: java/removeLowerCaseVowels.java
// remove lowercase vowels

public class removeLowerCaseVowels{

  public static String shortcut(String input) {
    return input.replaceAll("[aeiou]", "");
  }

  public static void main(String[] args){
    System.out.println(shortcut("hello and HELLO")); // "hll nd HELLO"
  }
}

File: c#/RemoveLowerCaseVowels.cs
// create function to remove lowercase vowels from string

using System.Text.RegularExpressions;

public class RemoveLowerCaseVowels
{
	public static string Shortcut(string str)
	{
		return Regex.Replace(str, @"[aeiou]", "");
	}

	public static void Main(string[] args)
	{
		System.Console.WriteLine(Shortcut("This is an elephant on it")); //Ths s n lphnt n t
	}
}